libs/shared/flagd-js-core/src/lib/flagd-js-core.ts

```diff
@@ -0,0 +1,133 @@
+import { Storage, StorageImpl } from './storage';
+import {
+  ErrorCode,
+  EvaluationContext,
+  FlagValue,
+  JsonValue,
+  ResolutionDetails,
+  StandardResolutionReasons,
+} from '@openfeature/server-sdk';
+
+/**
+ * Expose flag configuration setter and flag resolving methods.
+ */
+export class FlagdJSCore {
+  private _storage: Storage;
+
+  /**
+   * Construct with optional your own storage layer.
```

Comment:
```suggestion
   * Optionally construct with your own storage layer.
```

Reply:
Applied upstream, will update this draft PR once upstream is ready